### DISCUSSION TITLE
A few ideas based on my experiences with adadmin

### DISCUSSION THREAD

**Discussion by TheLlamaman** [giuliopons/adadmin#81] [Ideas]
**Campaign level Price and limitations**

At the moment you can set "Price and limitations" for a banner, such as "Total Impressions". It would be useful if you could alternatively set this at Campaign level.

My use case: On my website, I want to sell '50000 impressions' to advertisers. However I have multiple banner positions and sizes. Not all advertisers will want to use all positions. If you split the impressions per banner (e.g. 25000 for each of the 2 positions) then this is not great, because one position may use up the impressions quicker than another.

A former ad server I used (Revive) was capable of setting total impressions at campaign level.

**Banners which can go in multiple different positions**

I want my advertisers to be able to pick and choose which areas of a website that they have their banner. In adadmin, at the moment you can only assign one banner to one position.

If you have multiple different positions of the same size (e.g. 300x250), it would be useful to allow a banner to be assigned to multiple of these positions rather than one.

**More detailed statistics for banner positions**

It would be useful to have detailed stats on the Positions page, so you can see the CTR, clicks for each day. Similar to what you have on the Campaign and Banner pages.

**Comment by giuliopons** [comment]
Thank you for your suggestions. The "Banners which can go in multiple different positions" feature I now is needed, but it's too complex and I need some free time to work on it. I will think also about the other two.
